Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -961,9 +988,10 @@ namespace PlayerCommands
 		else
 		{
 			PrintUserCmdText(client, L"ERR Invalid parameters");
-			PrintUserCmdText(client, L"/base facmod [list|clear|add]");
+			PrintUserCmdText(client, L"/base facmod [list|clear|purge|add]");

```

Comment:
think we're going to need to do a better job of documenting clear vs. purge here

Reply:
documented as
"|  clear <index> - clear queue, which starts from second item in building order for factory module at <index>");
"|  purge <index> - clear only active recipe, which is current first device in building process for factory module at <index>");